src/components/paragraphs/stanford-card/card-paragraph.tsx

```diff
@@ -20,18 +22,22 @@ const CardParagraph = ({paragraph, ...props}: Props) => {
 
   const headerTagChoice = (behaviors.su_card_styles?.heading || "h2").split(".", 2);
   const headerTag = headerTagChoice[0]
-  const headerClasses = headerTagChoice[1]?.replace(".", " ").replace("su-font-splash", "text-m2 font-bold")
-
+  const headerClasses = headerTagChoice[1]?.replace(".", " ").replace("su-font-splash", "text-m2 font-normal mb-12") || "font-normal mb-12"
+  const cardVariant = behaviors.su_card_styles?.sum_card_variant;
+  const pillCardBgColor = behaviors.su_card_styles?.sum_card_bg_color_variant;
   const hideHeader = behaviors.su_card_styles?.hide_heading;
 
+  const Element = cardVariant ? PillCard : ImageCard
+
   return (
-    <ImageCard
+    <Element
       {...props}
       aria-labelledby={paragraph.suCardHeader ? paragraph.id : undefined}
       imageUrl={image?.url}
       imageAlt={image?.alt}
       videoUrl={videoUrl}
       isArticle={!!paragraph.suCardHeader}
+      bgColor={pillCardBgColor}
```

Comment:
Does this throw any logs since the image card doesn't take in this prop?

Reply:
Strangely there are no error throw for the image card 🤔

Reply:
> Warning: React does not recognize the `bgColor` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `bgcolor` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Reply:
I was able to see this in the console when I create a simple card, just add the headline field and then leave all other fields alone except choose a bg color.